docs/blog-marketing.md

```diff
@@ -0,0 +1,48 @@
+# Blog Documentation for Marketing
+
+The automated blog workflow has been designed to maintain the same workflow as any usual medium publication but we’ve added two small (but important) steps.
+
+Standard Workflow:
+
+- Author writes blog post and submits it for review to the YLD medium publication
+- Marketing will proof read and request any peer reviews required for the topic
+- Once the post has been approved, YLD publishes the blog normally.
+
+New additional steps
+
+- Update the canonical link
+- Add preview image (Not always required)
+
+## To update the canonical link
+
+**N.B. The new blog post will be live ~5-10mins after publishing depending on how long it takes for the task to be triggered**
+
+- Go to the new blog post, at the top right there is a settings/cog icon, click here and then click `Edit story`
+
+  ![edit post](./assets/edit.png)
+
+- Once you’re in editing mode click the three dots on the top right to open the dropdown. At the bottom of the dropdown list you’ll want to click `Customize canonical link`.
+
+  ![edit canonical](./assets/canonical.png)
+
+- Add the new blog post link from the yld.io site, hit save and you’re good to go!
+
+  ![edit canonical](./assets/canonical-url.png)
+
+## Add preivew image
+
+This step is only required if the blog published on medium has no images. Having images in blog posts is not critical to the new content but has two but It is advised that you encourage authors to add content but if not possible then these are the steps to follow to include the image.
+
+- Have the image you want to include saved to your computer. 1:1 ratio is preferred.
+- Go to Contentful and find the blog post you want to change.
+- In the blog post find the `Header Image` field
+- Add the new header image
+- Click publish and you're done!
+
+## FAQ
+
+Q: If I change content on medium will it update on yld.io?
+A: No, we only look for new urls in the RSS content and compare it to blog posts already published it on the CMS.
+
+Q: How do I change content on a blog post then? Do I have to update it twice?
+A: Yes, for this reason we encourage authors and proof readers to make sure everything is 100% ready before publishing but it can be easy to miss things sometimes! To change blog post content on CMS you'll need to write it in [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). Try not to change too much!
```

Comment:
Is deleting the post and triggering a re-export somehow an option?

Reply:
Yes definitely an option but would require manual triggering, will include